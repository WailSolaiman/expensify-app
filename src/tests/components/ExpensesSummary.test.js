import React from 'react';
import { shallow } from 'enzyme';
import { ExpensesSummary } from '../../components/ExpensesSummary';
import expenses from '../fixtures/expenses';

test('should render ExpensesSummary with no expenses correctly', () => {
    const expensesEmpty = [];
    const wrapper = shallow(<ExpensesSummary expenseCount={0} expensesTotal={0} />);
    expect(wrapper).toMatchSnapshot();
});

test('should render ExpensesSummary with expenses correctly', () => {
    const wrapper = shallow(<ExpensesSummary expenseCount={5} expensesTotal={14500} />);
    expect(wrapper).toMatchSnapshot();
});
