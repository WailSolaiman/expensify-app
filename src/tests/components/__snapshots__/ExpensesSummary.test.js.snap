// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpensesSummary with expenses correctly 1`] = `
<div>
  <h1>
    Viewing 5 expenses totaling $145.00
  </h1>
</div>
`;

exports[`should render ExpensesSummary with no expenses correctly 1`] = `
<div>
  <h1 />
</div>
`;
